src/app/nft-select-serial-number/nft-select-serial-number.component.html

```diff
@@ -57,19 +57,21 @@
     </div>
     <div *ngIf="includeColumn('high')" class="col-4 mb-0px d-flex align-items-center">
       <div>
-        <div class="d-lg-inline-block d-block">{{ globalVars.nanosToDeSo(nft.HighestBidAmountNanos) }} DESO </div>
-        <div class="text-grey7 d-lg-inline-block d-block">( ~{{globalVars.nanosToUSD(nft.HighestBidAmountNanos, 2) }})</div>
+        <div class="d-lg-inline-block d-block">{{ globalVars.nanosToDeSo(nft.HighestBidAmountNanos) }} DESO</div>
+        <div class="text-grey7 d-lg-inline-block d-block">
+          ( ~{{ globalVars.nanosToUSD(nft.HighestBidAmountNanos, 2) }})
+        </div>
       </div>
     </div>
     <div *ngIf="includeColumn('min')" class="col-5 mb-0px d-flex align-items-center">
       <div>
-        <div class="d-lg-inline-block d-block">{{ globalVars.nanosToDeSo(nft.MinBidAmountNanos) }} DESO </div>
+        <div class="d-lg-inline-block d-block">{{ globalVars.nanosToDeSo(nft.MinBidAmountNanos) }} DESO</div>
         <div class="text-grey7 d-lg-inline-block d-block">(~{{ globalVars.nanosToUSD(nft.MinBidAmountNanos, 2) }})</div>
       </div>
     </div>
     <div *ngIf="includeColumn('buyNow')" class="col-5 mb-0px d-flex align-items-center">
       <div>
-        <div class="d-lg-inline-block d-block">{{ globalVars.nanosToDeSo(nft.BuyNowPriceNanos) }} DESO </div>
+        <div class="d-lg-inline-block d-block">{{ globalVars.nanosToDeSo(nft.BuyNowPriceNanos) }} DESO</div>
```

Comment:
should the space still be there?

Reply:
I took a look at this UI. It looks okay without the space to be honest, but I think the space was intentional. You can see it in this screen shot, the space between DESO and the parens
<img width="530" alt="Screenshot 2023-04-10 at 7 18 20 PM" src="https://user-images.githubusercontent.com/7357287/231038580-06a35a72-745c-41fc-883a-151427b479c6.png">